README.md

```diff
@@ -14,3 +14,73 @@ Currently this supports:
 * major releases only
 * all probes registered in separate files (histograms, scalars, events)
 * some select environment data points (more to come)
+
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br>
```

Comment:
I could be wrong, but at least in GitHub *comments*, the `<br />` isn't needed for a non-paragraph-creating linebreak.

Reply:
Oh, this text is generated by CRA. Meh. Maybe not worth worrying about, then.